### PR TITLE
Notes for 2.0.0 [8529]

### PR DESCRIPTION
**Pull request by MiguelCompany**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/434/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/435/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/436/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/437/)

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -1,43 +1,38 @@
-Version 1.10.0
-==============
+Version 2.0.0
```
Comment (EduPonz) [inline docs/notes/notes.rst:1]: The notes for the last version could already be in their own file which is then included here. If you do that, make sure to rename `previous_versions` directory to something like `versions`.

File: docs/notes/notes.rst [hunk 2]
```diff
@@ -47,11 +42,13 @@ If you are upgrading from a version older than 1.10.0, regenerating the code is
 Previous versions
 -----------------
```
Comment (EduPonz) [inline docs/notes/notes.rst:43]: I'd put this title at the same level as the last version, i.e. `=====`. As it is now, on the toc tree, the "previous versions" section is placed inside the 2.0.0 version, it looks strange to me.

File: docs/notes/previous_versions/v1.8.4.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Version 1.8.4
+^^^^^^^^^^^^^
+
+This release adds the following **feature**:
```
Comment (EduPonz) [inline docs/notes/previous_versions/v1.8.4.rst:4]:
```suggestion
This release adds the following **features**:
```

File: docs/notes/previous_versions/v1.8.4.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+* XML profiles for `requester` and `replier`
+
+It also has the following **important bug fixes**:
+
+* Solved an issue when recreating a publishing participant with the same GUID (either on purpose or by chance)
+* Solved issue #1042, where a publisher could block on `write` for a long time when, after a large number of samples
```
Comment (EduPonz) [inline docs/notes/previous_versions/v1.8.4.rst:11]: There could be a link to the issue

File: docs/notes/notes.rst [hunk 5]
```diff
@@ -1,43 +1,38 @@
...
-* Fixed local host communications.
-* Correctly trimming server history.
-* Fixed backup server operation.
-* Fixed timing issues.
+* Support for OpenSSL 1.1.1d and higher
+* Support for latest versions of gtest
+* Support for FreeBSD
+* Fault tolerance improvements to Shared Memory transport
+* Fixed segfault when no network interfaces are detected
+* Correctly ignoring length of PID_SENTINEL on parameter list
```
Comment (EduPonz) [inline docs/notes/notes.rst:33]:
```suggestion
* Correctly ignoring length of `PID_SENTINEL` on parameter list
```

File: docs/notes/notes.rst [hunk 6]
```diff
@@ -1,43 +1,38 @@
...
-* Transport API refactored to support locator iterators
-* Added subscriber API to retrieve info of first non-taken sample
-* Added parameters to fully avoid dynamic allocations
-* History of built-in endpoints can be configured
-* Bump to FastCDR v1.0.13.
-* Bump to Fast-RTPS-Gen v1.0.4.
-* Require CMake 3.5 but use policies from 3.13
+* eClock API, which was deprecated on v1.9.1, has been removed
+* `eprosima::fastrtps::rtps::RTPSDomain::createParticipant` methods now have an additional first argument `domain_id`
+* Field `domainId` has been removed from `eprosima::fastrtps::rtps::RTPSParticipantAttributes` and added to
```
Comment (EduPonz) [inline docs/notes/notes.rst:8]:
```suggestion
* Data member `domainId` has been removed from `eprosima::fastrtps::rtps::RTPSParticipantAttributes` and added to
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/438/)

File: docs/notes/notes.rst [hunk 7]
```diff
@@ -1,43 +1,38 @@
...
-* Several data races and ABBA locks fixed.
-* Reduced CPU and memory usage.
-* Reduced flakiness of liveliness tests.
-* Allow for more use cases on performance tests.
+This release adds the following **features**:
 
-Several bug fixes on discovery server:
+* Added support for register/unregister/dispose instance
+* All the functionality offered by the classes in the Fast RTPS API can now be performed
+  by equivalent DDS compliant APIs
```
Comment (EduPonz) [inline docs/notes/notes.rst:22]:
```suggestion
* Added DDS compliant API. This new API exposes all the functionality of the Publisher-Subscriber Fast RTPS API adhering to the `Data Distribution Service (DDS) version 1.4 specification <https://www.omg.org/spec/DDS/1.4>`_
```

File: docs/notes/notes.rst (hunk 1 above)
Comment (MiguelCompany) [reply docs/notes/notes.rst:1]: I thought about that, but the title level is different for current and previous versions, so I left it this way.

File: docs/notes/previous_versions/v1.8.4.rst (hunk 3 above)
Comment (MiguelCompany) [reply docs/notes/previous_versions/v1.8.4.rst:4]: It is only one feature. Should we use plural anyways?

File: docs/notes/previous_versions/v1.8.4.rst (hunk 4 above)
Comment (MiguelCompany) [reply docs/notes/previous_versions/v1.8.4.rst:11]: Looking at the rest of the release notes, we have never referenced issues directly, so I am changing this to `an issue`

File: docs/notes/previous_versions/v1.8.4.rst (hunk 3 above)
Comment (EduPonz) [reply docs/notes/previous_versions/v1.8.4.rst:4]: I thought about it. The suggestion was more for consistency, but singular also makes sense

File: docs/notes/notes.rst (hunk 1 above)
Comment (EduPonz) [reply docs/notes/notes.rst:1]: Sure, is not a big deal to create the file afterwards anyways

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/439/)

**Pull request merged**